doxygen/dox/cookbook/Presets.dox

```diff
@@ -11,7 +11,7 @@ You want to build HDF5 with CMake, but there are so many options to consider.
 \par Solution
 CMake introduced presets in version 3.19. HDF Group provides a
 <a href="https://\SRCURL/CMakePresets.json"><tt>CMakePresets.json</tt></a>,
-requiring CMake 3.24 or higher,
+requiring CMake 3.25 or higher,
```

Comment:
This will quickly become outdated and will most likely be overlooked once it gets updated elsewhere.

Reply:
But they are hard minimum versions - when changing CMake versions then they can be eliminated.